data/saas/config/google_analytics_config.yml

```diff
@@ -92,6 +92,7 @@ saas_config:
     opt_out:
       method: POST
       path: /analytics/v3/userDeletion/userDeletionRequests:upsert
+      skip_missing_param_values: true
```

Comment:
This is a patch for the google analytics connector you reviewed previous @adamsachs - wanted to get your thoughts on this approach. Also open to better names!